source/documentation/deploying_services/elasticsearch.md

```diff
@@ -140,7 +140,7 @@ where `SERVICE_NAME` is a unique descriptive name for this service instance, and
 cf update-service my-elasticsearch-service -p small-ha-7.x
 ```
 
-The plan upgrade will start immediately and finish within an hour. You can check the status of the upgrade by running `cf services`.
+The plan upgrade will start immediately and finish within an hour, except for `large` and `xlarge` plans which may take over an hour. You can check the status of the upgrade by running `cf services`.
```

Comment:
Something sounds wrong about this.
I wonder if this would be better?

"The plan upgrade will start immediately and should finish within an hour. `large` and `xlarge` plans may take longer, depending on the amount of data stored."

Also, do we know if the service available while the upgrade is in process?

Reply:
Yes, I like your version better.